library/neo4j

```diff
@@ -9,12 +9,22 @@ Maintainers: Ben Butler-Cole <ben@neotechnology.com> (@benbc),
              Steven R Baker <steven.baker@neotechnology.com> (@srbaker),
              Praveena Fernandes <praveena.fernandes@neotechnology.com> (@praveenag)
 
-Tags: 3.1.1, 3.1, latest
+Tags: 3.1.2, 3.1, latest
+GitRepo: https://github.com/neo4j/docker-neo4j-publish.git
+GitCommit: 45fd4b39e21d47b9d57b9c891ca4e9f6f66534ed
+Directory: 3.1.2/community
+
+Tags: 3.1.2-enterprise, 3.1-enterprise, enterprise
+GitRepo: https://github.com/neo4j/docker-neo4j-publish.git
+GitCommit: 45fd4b39e21d47b9d57b9c891ca4e9f6f66534ed
+Directory: 3.1.2/enterprise
+
+Tags: 3.1.1
 GitRepo: https://github.com/neo4j/docker-neo4j-publish.git
 GitCommit: 6917c725e1b82ae5ee5ba2e5b26980101e9a3e35
 Directory: 3.1.1/community
 
-Tags: 3.1.1-enterprise, 3.1-enterprise, enterprise
+Tags: 3.1.1-enterprise
```

Comment:
Yes it is. We've had this discussion before as can be seen in the comment at the top of the file:

```
# Old point releases are included in this library file so that users
# are not forced to upgrade to the latest point releases in order to
# get security fixes to the base image. See
# https://github.com/docker-library/official-images/issues/1864 for a
# discussion of this point.
```

Maybe we can have a think about how many versions to keep for rebuilding however. I wonder if it still makes sense for `2.3.0` to be included for example.